backend/app/controllers/products.js

```diff
@@ -1208,11 +1190,26 @@ module.exports = {
             yield * checkProductData(req);
             if (parseInt(req.body.status) === 1) { // if status changed to 'STARTED'
                 var product = yield * common.getEntity(req, req.params.id, Product, 'id');
-                //TODO: Pull survey here from survey service and check
-                // var survey = yield * common.getEntity(req, product.surveyId, Survey, 'id');
-                // if (survey.isDraft) {
-                //     throw new HttpError(403, 'You can not start the project. Survey have status `in Draft`');
-                // }
+
+                //Check survey service is the survey exist in there
+                const path = 'surveys/';
+
+                const requestOptions = {
+                    url: config.surveyService + path + req.body.surveyId,
+                    method: 'GET',
+                    headers: {
+                        'authorization': req.headers.authorization
+                    }
+                };
+                request(
+                    requestOptions,
+                    function(err, response, body) {
+                        if (response.statusCode == 200) {
+                            if (body.status == 'draft')
+                                throw new HttpError(403, 'You can not start the project. Survey have status `in Draft`');                        }
```

Comment:
I need to change the error code here to be what is returned from the response `response.statusCode`